### PR TITLE
[ICP-12761] Add Chime event to installed, delete old metadata name from definition.

### PR DESCRIPTION
**Pull request by MGoralczykS**
@greens @dkirker @PKacprowiczS @ZWozniakS @KKlimczukS @MWierzbinskaS 
Could you review this?

### CODE REVIEW

**Review by greens — APPROVED** [review]

**Comment by greens** [comment]
@dkirker can you merge this after you've reviewed and merged the metadata?

File: devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy
Comment (dkirker) [inline devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy:17]: One thing I would recommend checking is installing a device with the DTH as it currently is in production (maybe best to self-publish so that you aren't changing the device type through the edit device) and then self-publishing this change to make sure that the new vid is used. I *think* on the DTH level there aren't problems, but I seem to recall issues with deleting a vid from a DTH where even if we were just relying on the DTH name we still had to declare it so that it would update for devices that were already installed.

Comment (dkirker) [reply devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy:17]: If that doesn't seem to be an issue then we should be fine to merge this.

Comment (MGoralczykS) [reply devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy:17]: @dkirker , @greens 
I left a comment under metadata MR. 
Should we use separately metadata for Siren and Doorbell?
I mean for Siren metadata with Alarm (generic-siren-12) capability and for a Doorbell with Chime.

Comment (MGoralczykS) [reply devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy:17]: @dkirker ping

Comment (dkirker) [reply devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy:17]: @MGoralczykS I haven't had a chance to look over the implications of either route. I'll have a look by Friday.

Comment (MGoralczykS) [reply devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy:17]: @dkirker any updates :)?

Comment (dkirker) [reply devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy:17]: @MGoralczykS I'm impartial to either way. Since we implement both devices the same way, and they are the same actual device but with a different fingerprint, it might make sense to just use the same metadata. If you want another opinion, you can check with product.

Comment (greens) [reply devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy:17]: @dkirker I think it's fine, then. Can you verify the metadata is out there and merge?

**Review by dkirker — APPROVED** [review]

**Pull request merged**
